content/chainguard/chainguard-images/retrieve-image-sboms.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "How to Retrieve SBOMs for Chainguard Images"
+linktitle: "Retrieve an Image's SBOM"
+type: "article"
+description: "A brief tutorial on how to use Cosign to retrieve Chainguard Image SBOMs."
+date: 2023-11-17T11:07:52+02:00
+lastmod: 2023-11-17T11:07:52+02:00
+draft: false
+tags: ["Conceptual", "Chainguard Images", "SBOM"]
+images: []
+menu:
+  docs:
+    parent: "chainguard-images"
+weight: 710
+toc: true
+---
+
+
+Chainguard Images contain only the minimum number of packages needed to use the software they contain. The purpose of this is to reduce the image's attack surface and minimize the risk that CVEs will impact software that depends on these container images. 
+
+Even though they contain the minimum number of packages, there may come a time when you want to know exactly what's running inside of a certain Chainguard Image. For this reason, we include a signed SBOM with each image in the form of a [software attestation](https://slsa.dev/attestation-model).
+
+[Cosign](/open-source/sigstore/cosign/an-introduction-to-cosign/) — a part of the Sigstore project — supports software artifact signing, verification, and storage in an [OCI (Open Container Initiative)](/open-source/oci/what-is-the-oci/) registry, as well as the retrieval of said artifacts. This tutorial outlines how you can use the `cosign` command to retrieve a Chainguard Image's SBOM. 
+
+
+## Prerequisites
+
+In order to follow this guide, you'll need the following installed on your local machine:
+
+* **Cosign** — to retrieve SBOMs associated with Chainguard Images, check out [our guide on installing Cosign](/open-source/sigstore/cosign/how-to-install-cosign/) to configure it.
+* **jq** — to process JSON, visit the [jq downloads](https://jqlang.github.io/jq/download/) page to set it up.
+
+
+## Using Cosign to retrieve an Image's SBOM
+
+Cosign includes a `download` command that allows you to retrieve a Chainguard Image's attestation over the command line. To do so, you would use this command with syntax like the following.
+
+```shell
+cosign download attestation \ 
+cgr.dev/chainguard/$IMAGE | jq -r .payload | base64 -d | jq .predicate
+```
+
```

Comment:
Could you include an example image here? It would just make it easier for someone to copy / paste